lib/OpenQA/WebAPI/Controller/API/V1/Iso.pm

```diff
@@ -604,7 +607,18 @@ sub create {
         }
     }
 
-    my $scheduled_jobs     = $self->schedule_iso(\%params);
+    my $scheduled_jobs = $self->schedule_iso(\%params);
+    my $error          = $scheduled_jobs->{error};
+    $self->render(
```

Comment:
Looks like this should be `return $self->render(`.

Reply:
Yes - I have already found this. That is the reason for the 500 in the scheduler API test.